core/package.json

```diff
@@ -8,7 +8,7 @@
     "@sveltejs/vite-plugin-svelte": "^2.5.3",
     "chai": "^4.3.6",
     "css-tree": "^2.3.1",
-    "fundamental-styles": "0.33.2",
+    "fundamental-styles": "^0.37.0",
```

Comment:
Please remove the `^` because it happens that this lib has breaking changes in minor releases.